### ISSUE TITLE
### <span aria-hidden="true">❌</span> Deploy Preview for *pedometria* failed.

### ISSUE THREAD

**Issue by DavidOPontes** [samuel-rosa/pedometria.org#14]
### <span aria-hidden="true">❌</span> Deploy Preview for *pedometria* failed.

|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | bf1564ad6fd60107da17892e8d841d7def7bfabe |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/sites/pedometria/deploys/654defb658d7d50009f2be15 |

_Originally posted by @netlify[bot] in https://github.com/Laboratorio-de-Pedometria/pedometria.org/issues/13#issuecomment-1805331371_
